services/121-service/test/payment/do-payment-fsp-visa-debit/do-payment-fsp-visa-debit-error.test.ts

```diff
@@ -323,6 +329,13 @@ describe('Do failing payment with FSP Visa Debit', () => {
     );
 
     expect(doPaymentResponse.status).toBe(HttpStatus.BAD_REQUEST);
+    // Check if both properties are mentioned in the error message
```

Comment:
the test fails. I think you should change add JSON.stringify  this to 

```
expect(JSON.stringify(doPaymentResponse.body)).toContain(
      FinancialServiceProviderConfigurationProperties.coverLetterCode,
    );
```

Reply:
I added .message instead